cli/src/semgrep/__main__.py

```diff
@@ -5,7 +5,7 @@
 
 
 def main() -> int:
-    cli()
+    cli(prog_name='semgrep')
```

Comment:
maybe add a comment like # this code is not run anymore from cli/bin/semgrep but from cli/bin/pysemgrep (exec'ed from cli/bin/semgrep) because of osemgrep, hence the prog_name adjustment to avoid listing 'pysemgrep for arv0'